### DISCUSSION TITLE
Read-only mode

### DISCUSSION THREAD

**Discussion by ntarocco** [inveniosoftware/product-rdm#171] [Ideas]
This feature request has been created after the implementation of @max-moser

Read-only mode can be used to implement a conditional read-only mode, e.g. enabled temporarily to make system migrations smoother.

### Pull requests
- https://github.com/inveniosoftware/invenio-vocabularies/pull/231
- https://github.com/inveniosoftware/invenio-requests/pull/293
- https://github.com/inveniosoftware/invenio-communities/pull/830
- https://github.com/inveniosoftware/invenio-rdm-records/pull/1120
- https://github.com/inveniosoftware/invenio-records-permissions/pull/86
- https://github.com/inveniosoftware/invenio-users-resources/pull/57
- https://github.com/inveniosoftware/invenio-drafts-resources/pull/209

**Comment by wgresshoff** [comment]
Hey,
what happened to this idea? We consider a (simple) module into place to stop accessing the repository but we guess it wouldn't stop users from calling the API. So we are still interested!
